docs/config_options.md

```diff
@@ -143,6 +143,7 @@ If you define these options you will enable the associated feature, which may in
   * Breaks any Tap Toggle functionality (`TT` or the One Shot Tap Toggle)
 * `#define LEADER_TIMEOUT 300`
   * how long before the leader key times out
+    * If you're having issues finishing the sequence before it times out, you may need to increase the timeout setting. Or you may want to switch to the `LEADER_PER_KEY_TIMING` option, which resets the timeout after each key is tapped. 
```

Comment:
Here are some things I would change. First I think there is no need to use so many may and we can use a more active style. Also, saying switch does not makes sense to me because you are not switching the config, you are adding an extra configuration option. How good does this sound:


If you're having issues finishing the sequence before it times out you can try to increase the. Alternatively you can use the `LEADER_PER_KEY_TIMING` option, which resets the timeout after each key is tapped.